### ISSUE TITLE
configure issue of netcdf-fortran with intel compiler in Ubunutu 22.04 .

### ISSUE THREAD

**Issue by Nagasapa** [Unidata/netcdf-fortran#436]
I am trying to compile netcdf-fortran-4.6.1 with intel-compilers(2024) in ubuntu-22.04 the following commands i have used in the build directory- **"../configure --prefix=/opt/netcdf-ifx/4.6.0 CPPFLAGS="-DNDEBUG -DpgiFortran -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -I/opt/zlib/include -I/home/nagarjuna/Desktop/HDF5_Intel/myhdfstuff/build/HDF_Group/HDF5/1.14.3/include -I/opt/netcdf_c1/include" LDFLAGS="-L/opt/zlib/lib -Wl,-rpath=/opt/zlib/lib -L/home/nagarjuna/Desktop/HDF5_Intel/myhdfstuff/build/HDF_Group/HDF5/1.14.3/lib -Wl,-rpath=/home/nagarjuna/Desktop/HDF5_Intel/myhdfstuff/build/HDF_Group/HDF5/1.14.3/lib -L/opt/netcdf_c1/lib -Wl,-rpath=/opt/netcdf_c1/lib " --enable-large-file-tests --with-pic --enable-static --enable-shared "** and  used these compilers export CC=icx
export FC=mpiifx
export F90=mpiifx
export F77=mpiifx
export CXX=icpx
export MKL_TARGET_ARCH=intel64..Got error like this - "**configure: error: cannot compute sizeof (off_t)
See `config.log' for more details**". How this issue can be resolved?

Cheers !!

**Comment by jshn9515** [comment]
This problem could occur if you used different compilers between all the dependencies and NetCDF-C.

**Comment by Nagasapa** [comment]
yes i have built zlib ,hdf5, netcdf-c with intel compilers(2024).

**Comment by WardF** [comment]
Can you provide the `config.log`? This sometimes occurs when there is a problem linking the test program autotools uses to determine sizeof various variable types.
